packages/rum-core/src/domain/view/viewMetrics/startInitialViewMetricsTelemetry.ts

```diff
@@ -35,7 +35,7 @@ export function startInitialViewMetricsTelemetry(lifeCycle: LifeCycle, telemetry
     // performance issues early in the page load process, and using LCP-at-page-load is a
     // good fit for that use case, but it's important to be aware that this is not
     // necessarily equivalent to the normal LCP metric.
-
+    // monitor-until: forever
     addTelemetryMetrics(TelemetryMetrics.INITIAL_VIEW_METRICS_TELEMETRY_NAME, {
```

Comment:
@sethfowler-datadog I put forever for all the new metrics but let me know if some could be temporary